python/mirascope/llm/providers/provider_registry.py

```diff
@@ -15,6 +15,7 @@
 # These providers will be automatically registered on first use
 DEFAULT_AUTO_REGISTER_SCOPES: dict[str, ProviderId] = {
     "anthropic/": "anthropic",
+    "anthropic-beta/": "anthropic-beta",
```

Comment:
Hmm I'm not sure about putting this in default_auto_register_scopes since I don't think users will get any benefit from it? Though I guess there is also no harm in it either. I had been imagining that we'd leave it out and just manually register it.